### PR TITLE
yet another compilation fix 3

### PR DESCRIPTION
**Pull request by mtasaka**
Build error fix part3 from me

### CODE REVIEW

**Comment by turboscholz** [comment]
Great! Thanks. As I have only little time today I can not contribute with own fixes. But you rock it! :+1:

File: src/gnome-cmd-advrename-lexer.ll [hunk 1]
```diff
@@ -160,6 +160,7 @@ tag_name    {ape}|{audio}|{doc}|{exif}|{file}|{flac}|{id3}|{image}|{iptc}|{pdf}|
                                     case 'n' : p->type = NAME;            break;
                                     case 'N' : p->type = FULL_NAME;       break;
                                     case 'p' : p->type = PARENT_DIR;      break;
+                                    default: break;
```
Comment (turboscholz) [inline src/gnome-cmd-advrename-lexer.ll]: The warning, which is still printed after your modification, seems to be a known bug in flex: https://github.com/westes/flex/issues/103

Comment (mtasaka) [inline src/gnome-cmd-advrename-lexer.ll]: > The warning, which is still printed after your modification, seems to be a known bug in flex: westes/flex#103

Did you regenerate src/gnome-cmd-advrename-lexer.cc (i.e. once remove src/gnome-cmd-advrename-lexer.cc "explicitly" and make again)?  Note that currently "make clean" does not clean up src/gnome-cmd-advrename-lexer.cc, you have to "rm" this explicitly.

Also, workaround for flex side is committed: see ff637f03ce74aa06113aa3b12d12cb071b336ada

Comment (turboscholz) [inline src/gnome-cmd-advrename-lexer.ll]: Ah, thanks. :+1: Removing src/gnome-cmd-advrename-lexer.cc manually did the trick. I did a make clean before but did not realized that it was not removed...

Comment (turboscholz) [inline src/gnome-cmd-advrename-lexer.ll]: I added the file to CLEANFILES so that it is removed by "make clean" in commit 1c932e2.

**Pull request merged**